tutorials/01_introductory/sample_plots.py

```diff
@@ -1,19 +1,20 @@
-.. _matplotlibscreenshots:
-
-**********************
-Screenshots
-**********************
+"""
+==========================
+Sample plots in Matplotlib
+==========================
 
 Here you'll find a host of example plots with the code that
 generated them.
 
+.. _matplotlibscreenshots:
+
 Simple Plot
 ===========
 
 Here's a very basic :func:`~matplotlib.pyplot.plot` with text labels:
```

Comment:
I like to avoid the word basic/simple/etc. in tutorials. What about "This is how to create a line plot with text labels"